js/main.js

```diff
@@ -196,14 +196,15 @@ Site.Product = {
 };
 
 Site.Layout = {
-  windowWidth: $(window).width(),
-  windowHeight: $(window).height(),
   init: function() {
     var _this = this;
 
+    _this.windowWidth = $(window).width();
+    _this.windowHeight = $(window).height();
```

Comment:
I can't see that these values are updated on resize, but might not be clear in the github diff view